### DISCUSSION TITLE
The ot1.3 standard normalizes the border router. Is there a specific application description

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#8121] [Q&A] [answered]
Please consult the experts：
The ot1.3 standard normalizes the border router. Is there a specific application description，For example, the incoming and outgoing processes of data, and the protocols of external applications

**Comment by jwhui** [comment, marked as answer]
Some useful documents:

- [Thread Border Router](https://www.threadgroup.org/Portals/0/documents/support/ThreadBorderRouterWhitePaper_07192022_4001_1.pdf) white paper
- [Thread 1.3.0 features](https://www.threadgroup.org/Portals/0/documents/support/Thread1.3.0WhitePaper_07192022_3990_1.pdf) overview
- [Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery](https://openthread.io/codelabs/openthread-border-router) code lab

**Reply by tuopu** [reply]
thank yourThank you for your guidance and reply.
